package.json

```diff
@@ -40,7 +41,9 @@
     "lint:fix": "yarn run lint --fix",
     "format": "prettier --write '**/*.{json,md,scss,yaml,yml}'",
     "format:check": "prettier --check '**/*.{json,md,scss,yaml,yml}'",
-    "update-api": "./scripts/update-api.sh ./src/common/api/swagger.yaml ./src/common/api/types.ts"
+    "update-api": "./scripts/update-api.sh ./src/common/api/swagger.yaml ./src/common/api/types.ts",
+    "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c ./scripts/i18next-parser.config.js",
+    "cleanTranslationFile": "./scripts/clean-translation-files.sh"
```

Comment:
I'd put `./scripts/clean-translation-files.sh` as a part of `i18n`
unless there is a usecase to run `i18n` without `cleanTranslationFile` ?

Reply:
Just wondering
can't we do the opposite of what we're doing now ?

Currently we do

Reply:
Just wondering
can't we do the opposite of what we're doing now ?

Currently we do:
1. in the code
```
t('ui:No host is selected.')
```

2. which generates locales
"ui:No host is selected.": "ui:No host is selected."

3. then we clean up the values
"ui:No host is selected.": "No host is selected."


But we could do
1. in the code (no prefix)
```
t('No host is selected.')
```

2. which generates locales
"No host is selected.": "No host is selected."

3. then we add prefix to keys
"ui:No host is selected.": "No host is selected."

Reply:
It's true. Always run together